src/database.rs

```diff
@@ -89,6 +103,23 @@ impl<'a> Database<'a> {
         ))
     }
 
+    /// Execute a command on the database.
+    /// Automates the process of getting the next batch from getMore
+    /// and parses the batch so only the result documents are returned.
+    /// I am unsure of the best practices of when to use this or the CRUD function.
```

Comment:
You can link to the new types here to make sure the docs are easily navigatable. I think the last line could be written differently. If we don't know remove it? What's automated about it, it returns a type that lets you access it no?

Reply:
* I use db.command because I want the flexibility to submit couplex queries easily using a consistent interface, but I have not done enough research to know what mongo recommends.  (I think they prefer you use the CRUD functions.). For someone looking to do something quickly I didn't want them to blindly use this function without being aware of these concerns.

* on the automation comment--I can use another work, but as it says, it prevents the user from having to manually issue a getMore command back to the db.